protocol/errors.json

```diff
@@ -61,6 +61,26 @@
 	"50002": "internal connection error",
 	"50003": "timeout error",
 
+	/* frontend connection-related */
+	"55000": "internal error (uncategorised)",
+
+	/* frontend connection-related */
```

Comment:
> What about auth failures?

We could add a category for auth but the internal errors I can think of would fit into other categories: eg persistence failure for persistent token; persistence failure for redis lookup failure; core rpc error for failure to instance an app client.